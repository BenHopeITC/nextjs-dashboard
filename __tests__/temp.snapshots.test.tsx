import { render } from '@testing-library/react'
import Home from '@/app/temp/home'
 
it('renders homepage unchanged', () => {
  const { container } = render(<Home />)
  expect(container).toMatchSnapshot()
})