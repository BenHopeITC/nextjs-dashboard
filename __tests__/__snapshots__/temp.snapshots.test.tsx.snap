// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <h1>
      Home
    </h1>
    <h3>
      Sub Home
    </h3>
    <a
      href="/about"
    >
      About
    </a>
  </div>
</div>
`;
